staging_docs/admin/learn/authentication.md

```diff
@@ -117,11 +117,222 @@ via the setting `TOKEN_EXPIRATION_TIME`. The default expiration time is `300` se
+Cross-Origin-Opener-Policy: same-origin
+Date: Fri, 17 May 2024 09:20:29 GMT
+Referrer-Policy: same-origin
+Server: nginx
+Strict-Transport-Security: max-age=15768000
+Vary: Accept
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+
+{
+    "access_token": <TOKEN>,
+    "expires_in": 300,
+    "issued_at": "2024-05-17T09:20:29.660984Z",
+    "token": <TOKEN>
+}
+
+
+$ http https://puffy.example.com/v2/library/azure/tags/list --auth-type=jwt --auth=<TOKEN>
+HTTP/1.1 200 OK
+Access-Control-Expose-Headers: Correlation-ID
+Allow: GET, HEAD, OPTIONS
+Connection: keep-alive
+Content-Length: 42
+Content-Type: application/json
+Correlation-ID: b944d8b6b1fc474eada8531776a2609b
+Cross-Origin-Opener-Policy: same-origin
+Date: Fri, 17 May 2024 09:33:04 GMT
+Docker-Distribution-Api-Version: registry/2.0
+Referrer-Policy: same-origin
+Server: nginx
+Strict-Transport-Security: max-age=15768000
+Vary: Accept
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-Registry-Supports-Signatures: 1
+
+{
+    "name": "library/azure",
+    "tags": [
+        "latest"
+    ]
+}
+```
+
+!!! note
+
+    If the repository is private, in order to pull content from it, one would need to provide basic auth
+    credentials while requesting the token. If no basic auth is provided, then the generated token is for
+    the anonymous user.
+    In order to push content to the registery a user always needs to be autenticated.
+
+```bash
+$ http 'https://puffy.example.com/token/?service=puffy.example.com&scope=repository:private/azure:pull' --auth-type=basic --auth=alice:wonderland
+```
+
+!!! note
+
+    Some registry endpoints like ``_catalog`` endpoint are not opened to anonymous/not logged in users and
+    require credentials provided during the token request. Anonymous token will still lead to 401 insufficient scope.
+
+```bash
+http 'https://puffy.example.com/token/?service=puffy.example.com&scope=registry:catalog:*' --auth-type=basic --auth=alice:wonderland
+```
+
+This token embeds permissions that allow to see only those repositories that Alice has access to. 
+
+```bash
+$ http https://puffy.example.com/v2/_catalog --auth-type=jwt --auth=<TOKEN>
+HTTP/1.1 200 OK
+Access-Control-Expose-Headers: Correlation-ID
+Allow: GET, HEAD, OPTIONS
+Connection: keep-alive
+Content-Length: 63
+Content-Type: application/json
+Correlation-ID: f4ea6c26bcb7462099c0d5757178f7ca
+Cross-Origin-Opener-Policy: same-origin
+Date: Fri, 17 May 2024 09:57:11 GMT
+Docker-Distribution-Api-Version: registry/2.0
+Referrer-Policy: same-origin
+Server: nginx
+Strict-Transport-Security: max-age=15768000
+Vary: Accept
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-Registry-Supports-Signatures: 1
+
+{
+    "repositories": [
+        "alice/azure",
+        "alice/openstack-cron"
+    ]
+}
+```
+
+If you are still unsure why you are geting 401 with the generated token, paste its payload to https://jwt.io/
+and it will help identify whether the token contains the needed access.
```

Comment:
```suggestion
If you are still unsure why you are getting HTTP 401 errors with a generated token, paste its payload to https://jwt.io/
and it will help you identify whether the token contains the needed access.
```